secant/Stubs/MockServices.swift

```diff
@@ -58,117 +58,82 @@ class MockMnemonicPhraseHandling: MnemonicSeedPhraseHandling {
 }
 
 class KeysPresentStub: KeyStoring {
+    var called = false
+    var returnBlock: () throws -> Bool
```

Comment:
Should we name these after what is being called and what is being returned?  Since there are multiple functions here it might be easier to read the tests if the naming was more associated with what it is you are testing.
i.e. `areKeysPresentCalled` etc.

Reply:
Since MockServices.swift is not used anywhere, removing it completely from the project